types/chec__commerce.js/types/product.d.ts

```diff
@@ -52,8 +64,9 @@ export interface Product {
     checkout: string;
     display: string;
   };
-  variants: Variant[];
+  extra_fields: any[];
+  variant_groups: Variant[];
```

Comment:
These are a list of variant groups now. We probably need to update those types to suit the latest API version. A quick summary would be:

Product's `variant_groups: VariantGroup[]`:

* id: string
* name: string
* meta: object|array ?
* created: int?
* updated: int?
* options: VariantOption[]

A Variant is a type, but it's not connected directly in product responses any more. Instead, there's a separate API to fetch variants for a product: https://commercejs.com/docs/sdk/products#get-variants. This API returns a list of Variants, and a Variant object looks like this:

* id: stirng
* sku: string
* description: string
* inventory: int?
* price: Price?
* is_valid: boolean
* invalid_reason_code: string
* meta: object|array ?
* created: int?
* updated: int?
* options: VariantOption[]
* assets: Asset[]

The `options` attribute in a variant object lists the variant options that make up this variant, for example if the product was "New season T-shirt", the variant might be "Small + Red" and the variant options would be "Small" and "Red".

Would you be keen to add in these new types?